### PR TITLE
14778 new registry page

### PR DESCRIPTION
**Pull request by BibianaC**
Filters are missing but would be good to already receive some feedback of what is done.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# 360Giving Registry
+
+Install dependencies:
+
+```
+$ python3 -m virtualenv -p $(which python3) .ve
```
Comment (robredpath) [inline README.md]: When I tried this, I got `/bin/python3: No module named virtualenv` , but I changed it to `python3 -m venv .ve` and it worked. I think the latter is the preferred python 3 way of doing this anyway.

File: registry/templates/registry.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+          use it easily and for free. You can use the table below to download datasets from these organisations.
+          You will see that each dataset has its own license attached to it. These licenses specify how the data
+          can be shared and used. The data in these spreadsheets is used and displayed by a growing number of
+          tools and websites. If you are looking for examples of what can be created using this data,
+          please visit the <a href="http://www.threesixtygiving.org/data/find-data/" target="_blank">Tools &amp; Projects</a>
+          page.
+        </p>
+        <p class="devs-info">
+          <strong>Developers:</strong> For a JSON feed of published datasets visit
+          <a href="http://data.threesixtygiving.org" target="_blank">http://standard.threesixtygiving.org/en/latest/getdata/</a>
```
Comment (robredpath) [inline registry/templates/registry.html]: We should update the contents of the link text as well as the link target

File: registry/templates/registry.html [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+          <div class="col-sm-3">
+            <div class="row align-items-start">
+              <a href="http://www.threesixtygiving.org/">
+                <img src="http://www.threesixtygiving.org/wp-content/themes/360giving/assets/img/logo-white.png" alt="360Giving" width="140" height="100">
+              </a>
+            </div>
+          </div>
+          <div class="col-sm-3">
+            <ul class="footer-info-col">
+              <li><a title="Contact" href="http://www.threesixtygiving.org/contact/">CONTACT</a></li>
```
Comment (robredpath) [inline registry/templates/registry.html]: We can remove this column

File: registry/templates/registry.html [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+            </ul>
+          </div>
+          <div class="col-sm-3">
+            <ul>
+              <li><a title="Privacy" href="http://www.threesixtygiving.org/privacy/">Privacy</a></li>
+              <li><a title="Terms &#038; Conditions" href="http://www.threesixtygiving.org/terms-conditions/">Terms &amp; Conditions</a></li>
+              <li><a title="Cookie Policy" href="http://www.threesixtygiving.org/cookie-policy/">Cookie Policy</a></li>
+              <li><a title="Take Down Policy" href="http://www.threesixtygiving.org/take-down-policy/">Take Down Policy</a></li>
+              <li><a title="License" href="https://creativecommons.org/licenses/by/4.0/">License</a></li>
+              <li><a href="https://www.logicdesign.co.uk" target="_blank">Website Design by Logic Design</a></li>
```
Comment (robredpath) [inline registry/templates/registry.html]: We can remove this link

File: registry/views.py [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+        if prefix not in data_by_prefix:
+            data_by_prefix[prefix] = get_prefix_data(data)
+
+        data_by_prefix[prefix]['grant'].append(get_grant_data(data))
+
+    return OrderedDict(sorted(data_by_prefix.items(), key=lambda x: x[1]['publisher']['name']))
+
+
+@app.route('/data/data-registry')
```
Comment (robredpath) [inline registry/views.py]: This should now be `/` as we're hosting this on its own server

File: README.md (hunk 1 above)
Comment (BibianaC) [reply README.md]: I will update it. Thanks.

File: registry/templates/registry.html (hunk 2 above)
Comment (BibianaC) [reply registry/templates/registry.html]: Good catch!

File: registry/templates/registry.html (hunk 3 above)
Comment (BibianaC) [reply registry/templates/registry.html]: Ok! I will delete it.

File: registry/templates/registry.html (hunk 4 above)
Comment (BibianaC) [reply registry/templates/registry.html]: Ok! I will remove it.

File: registry/views.py (hunk 5 above)
Comment (BibianaC) [reply registry/views.py]: ah yes. I will update it. Thanks

**Comment by robredpath** [comment]
This is great @BibianaC - although I did realise that I missed one (fairly large) thing in the review! We normally have functional tests for web stuff, using Selenium - please could you add some here? 

I've also made you an Owner - ping me and we can set up Travis for the repo together :)

**Pull request merged**